package.json

```diff
@@ -6,7 +6,7 @@
   "main": "index.js",
   "scripts": {
     "start": "probot run ./index.js",
-    "test": "mocha && xo"
+    "test": "mocha && standard"
```

Comment:
I usually put "standard" in a `"pretest"` script, like here:
https://github.com/hoodiehq/hoodie/blob/9bcd805fd01e905dd53b94c5ca547967cdd3f893/package.json#L73

Reply:
I didn't know npm had a `pretest` field! cool!

Reply:
oh yeah, you can use `pre<script name>` and `post<script name>` for any of the scripts defined in package.json, see https://docs.npmjs.com/misc/scripts

> Pre and post commands with matching names will be run for those as well (e.g. premyscript, myscript, postmyscript)

Reply:
My preference would be posttest. I would rather focus on getting the tests passing before worrying about the aesthetics of the code.